src/db/cass/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6
+
+WORKDIR /cass-app
+
+ENV NGINX_RTMP_IP=127.0.0.1
+
+ENV NGINX_RTMP_PORT=19350
+
+ENV CASSANDRA_CLUSTER_IPS="192.168.1.15"
+
+ENV CASSANDRA_CLUSTER_PORT="9043"
```

Comment:
I think you meant `9042`?

Reply:
I'm just testing it so it could be connected to any port.